tools/scripts/gen-env-files.sh

```diff
@@ -445,6 +445,38 @@ MEDIA_STREAM_TIMEOUT=60000" >> ./services/audio/audio.env
     echo "./services/audio/audio.env created"
 fi
 
+## Audio - Clip Producer
+if test -f "./services/audio/audio.env"; then
+    echo "./services/audio/atom.env exists"
+else
+echo \
+"KEYCLOAK_AUTH_SERVER_URL=http://host.docker.internal:$portKeycloak/auth/
+
+DB_URL=jdbc:postgresql://host.docker.internal:$portDatabase/$dbName
+DB_USERNAME=$dbUser
+DB_PASSWORD=$password
+
+KAFKA_LOGS_TOPIC=logs-audio
+
+KAFKA_BOOTSTRAP_SERVERS=host.docker.internal:$portKafkaBorkerAdvertisedExternal
+KAFKA_CLIENT_ID=audio-capture-01
+
+MAX_FAILED_ATTEMPTS=5
+
+DATA_SOURCE_ID=CBCKAM
```

Comment:
You're configuring the full datasource in the `.env`.  I think we are no longer doing this.  Most of this data is coming from the database now.

Reply:
I removed the redundant entries and created a section for the capture service.